### DISCUSSION TITLE
Missing Arduino ESP32 Docs

### DISCUSSION THREAD

**Discussion by pedrominatel** [espressif/arduino-esp32#6212] [Ideas]
Hi,

Please add here all the missing parts/chapters/sections/information or issues related to the documentation.

[Arduino ESP32 Documentation](https://docs.espressif.com/projects/arduino-esp32/en/latest/)

Thank you!

**Comment by VojtechBartoska** [comment]
Project for tracking Documentation tasks is available [**here**](https://github.com/espressif/arduino-esp32/projects/5)

**Comment by pedrominatel** [comment]
Can we lint issues here, @VojtechBartoska?

https://github.com/espressif/arduino-esp32/issues/6228

**Reply by VojtechBartoska** [reply]
why not and I will also add it to the Documentation project Roadmap.
